generic-types-traits/includes/2-define-shared-behavior-with-traits.md

```diff
@@ -0,0 +1,468 @@
+# Define shared behavior with traits
```

Comment:
@tilacog is there a good way this file can be broken up? It's too long for one unit. If you can split this into two, or even three units that would be better.

Reply:
Ok, I will do that.